docs/tutorial.md

```diff
@@ -0,0 +1,202 @@
+        "taper": {
+            "top": {"angle_in_deg": 0.0, "height_in_mm": 0.0},
+            "bottom": {"angle_in_deg": 0.0, "height_in_mm": 0.0},
+        },
+    },
+}
+
+coax_meta = {
+    "name": "C000RG1",
+    "type": "coax",
+    "production": {
+        "enrichment": {"val": 0.855, "unc": 0.015},
+    },
+    "geometry": {
+        "height_in_mm": 84,
+        "radius_in_mm": 38.25,
+        "borehole": {"radius_in_mm": 6.75, "depth_in_mm": 73},
+        "groove": {"depth_in_mm": 2, "radius_in_mm": {"outer": 20, "inner": 17}},
+        "pp_contact": {"radius_in_mm": 17, "depth_in_mm": 0},
+        "taper": {
+            "top": {"angle_in_deg": 45, "height_in_mm": 5},
+            "bottom": {"angle_in_deg": 45, "height_in_mm": 2},
+            "borehole": {"angle_in_deg": 0, "height_in_mm": 0},
+        },
+    }
+}
+```
+
+Now we can build all the logical volumes and place them in the world volume:
+
+```python
+# create logical volumes for the two HPGe detectors
+bege_l = hpges.make_hpge(bege_meta, name="BEGe_L", registry=reg)
+coax_l = hpges.make_hpge(coax_meta, name="Coax_L", registry=reg)
+
+# create a world volume
+world_s = pg4.geant4.solid.Orb("World_s", 20, registry=reg, lunit="cm")
+world_l = pg4.geant4.LogicalVolume(world_s, "G4_Galactic", "World", registry=reg)
+reg.setWorld(world_l)
+
+# let's make a liquid argon balloon
+lar_s = pg4.geant4.solid.Orb("LAr_s", 15, registry=reg, lunit="cm")
+lar_l = pg4.geant4.LogicalVolume(lar_s, "G4_lAr", "LAr_l", registry=reg)
+pg4.geant4.PhysicalVolume([0, 0, 0], [0, 0, 0], lar_l, "LAr", world_l, registry=reg)
+
+# now place the two HPGe detectors in the argon
+pg4.geant4.PhysicalVolume([0, 0, 0], [5, 0, -3, "cm"], bege_l, "BEGe", lar_l, registry=reg)
+pg4.geant4.PhysicalVolume([0, 0, 0], [-5, 0, -3, "cm"], coax_l, "Coax", lar_l, registry=reg)
+
+# finally create a small radioactive source
+source_s = pg4.geant4.solid.Tubs("Source_s", 0, 1, 1, 0, 2*pi, registry=reg)
+source_l = pg4.geant4.LogicalVolume(source_s, "G4_BRAIN_ICRP", "Source_L", registry=reg)
+pg4.geant4.PhysicalVolume([0, 0, 0], [0, 5, 0, "cm"], source_l, "Source", lar_l, registry=reg)
+```
+
+Note how we also created a small cylinder to represent a radioactive source
+later in the simulation.
+
+Now we can quickly visualize the result, still with *pyg4ometry*:
+
+```python
+# start an interactive VTK viewer instance
+viewer = pg4.visualisation.VtkViewerColoured(materialVisOptions={"G4_lAr": [0, 0, 1, 0.1]})
+viewer.addLogicalVolume(reg.getWorldVolume())
+viewer.view()
+```
+
+![Geometry visualization](img/tutorial-pyg4-view.jpg)
+
+By following instructions in the <project:./install.rst> section, you should
+have access to the `remage` executable. We are now ready to simulate some
+particle physics with it.
+
+Like any other Geant4-based application, we need to configure the simulation
+with a macro file. Standard Geant4 commands as well as custom commands (see the
+<project:./rmg-commands.rst>) are available.
```

Comment:
those `project:./` links do not work (at least on RTD, did not try locally)

Reply:
uff works locally...